h/static/scripts/util/api/index.ts

```diff
@@ -298,18 +298,28 @@ export type Pagination = {
   pageSize?: number;
 };
 
+export type CursorPagination = {
+  /** Return paginated items after this cursor */
+  after?: string;
+  /** Maximum number of items to return in response for a paginated API. */
+  pageSize?: number;
+};
```

Comment:
As we discussed yesterday, it is perfectly possible in principle to combine `after` with `pageSize` and `pageNumber`. I assume the backend does not allow this yet and that's why it is modeled this way here?